variables.md

```diff
@@ -304,12 +304,12 @@ PUBLIC_CLOUD_IMG_PROOF_EXCLUDE | string | "" | Tests to be excluded by img-proof
 PUBLIC_CLOUD_INSTANCE_TYPE | string | "" | Specify the instance type. Which instance types exists depends on the CSP. (default-azure: Standard_A2, default-ec2: t2.large )
 PUBLIC_CLOUD_LTP | boolean | false | If set, the run_ltp test module is added to the job.
 PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE | boolean | false | Do not remove the instance when the test fails.
-PUBLIC_CLOUD_PERF_COLLECT | boolean | false | Enable `boottime` measures collection, at end of `create_instance` routine.
-PUBLIC_CLOUD_PERF_DB_URI | string | "" | If set, the bootup times get stored in the influx database. (e.g. PUBLIC_CLOUD_PERF_DB_URI=http://publiccloud-ng.qe.suse.de:8086)
+PUBLIC_CLOUD_PERF_COLLECT | boolean | 1 | 0:disable, 1:enable `boottime` measures collection, at end of `create_instance` routine.
```

Comment:
```suggestion
PUBLIC_CLOUD_PERF_COLLECT | boolean | 1 | enables `boottime` measurement collection, at end of `create_instance` routine.
```

IMHO this is unnecessary verbosity

Reply:
ehm if someone prefers to use `empty` or undefined '',  willing to disable all, it will not happen obtaining default=1. This verbosity I considered necessary to avoid future misunderstanding

Reply:
and here I will agree with Felix , this is something obvious or should/can be added to ALL variables in this list

Reply:
done